[2024-12-15T00:42:08.121-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_price_volume_etl.retry_missing_tickers manual__2024-12-15T05:40:44.965634+00:00 [queued]>
[2024-12-15T00:42:08.125-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_price_volume_etl.retry_missing_tickers manual__2024-12-15T05:40:44.965634+00:00 [queued]>
[2024-12-15T00:42:08.125-0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2024-12-15T00:42:08.131-0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retry_missing_tickers> on 2024-12-15 05:40:44.965634+00:00
[2024-12-15T00:42:08.134-0500] {standard_task_runner.py:57} INFO - Started process 642 to run task
[2024-12-15T00:42:08.136-0500] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'daily_price_volume_etl', 'retry_missing_tickers', 'manual__2024-12-15T05:40:44.965634+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpxrti025q']
[2024-12-15T00:42:08.138-0500] {standard_task_runner.py:85} INFO - Job 327: Subtask retry_missing_tickers
[2024-12-15T00:42:08.158-0500] {task_command.py:416} INFO - Running <TaskInstance: daily_price_volume_etl.retry_missing_tickers manual__2024-12-15T05:40:44.965634+00:00 [running]> on host d5d7f84d2f8f
[2024-12-15T00:42:08.196-0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_price_volume_etl' AIRFLOW_CTX_TASK_ID='retry_missing_tickers' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T05:40:44.965634+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-15T05:40:44.965634+00:00'
[2024-12-15T00:42:12.944-0500] {multi.py:176} ERROR - 
1 Failed download:
[2024-12-15T00:42:12.944-0500] {multi.py:189} ERROR - ['N']: YFPricesMissingError('$%ticker%: possibly delisted; no price data found  (1d 2023-12-10 -> 2024-12-10)')
[2024-12-15T00:42:12.955-0500] {etl.py:136} INFO - All columns validated successfully.
[2024-12-15T00:42:12.957-0500] {logging_mixin.py:154} INFO - Number of records with out-of-range AdjClose: 0
[2024-12-15T00:42:12.958-0500] {logging_mixin.py:154} INFO - Number of records with out-of-range Date: 0
[2024-12-15T00:42:12.960-0500] {etl.py:189} INFO - Dropped 2 rows with missing values in ['ticker', 'date', 'adj_close', 'close', 'ret_1d'].
[2024-12-15T00:42:12.961-0500] {etl.py:222} WARNING - Missing tickers: ['N', 'e', 'n', 'o']
[2024-12-15T00:42:12.961-0500] {etl.py:223} INFO - Transformation pipeline completed successfully.
[2024-12-15T00:42:12.984-0500] {logging_mixin.py:154} INFO - Inserted 500 rows out of 500 records
[2024-12-15T00:42:12.985-0500] {python.py:194} INFO - Done. Returned value was: None
[2024-12-15T00:42:12.989-0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=daily_price_volume_etl, task_id=retry_missing_tickers, execution_date=20241215T054044, start_date=20241215T054208, end_date=20241215T054212
[2024-12-15T00:42:13.028-0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-15T00:42:13.036-0500] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
